app/uk/gov/hmrc/economiccrimelevyreturns/models/email/ReturnSubmittedEmailRequest.scala

```diff
@@ -27,8 +27,7 @@ final case class ReturnSubmittedEmailParameters(
   fyStartYear: String,
   fyEndYear: String,
   datePaymentDue: Option[String],
-  amountDue: String,
-  privateBetaEnabled: String
```

Comment:
This is not an `Option`. You can't remove it without changing the receiving service.